web_ui/tests/features/project-labels/project-labels.spec.ts

```diff
@@ -94,9 +94,14 @@ test.describe('Labels editor', () => {
                         ],
                     },
                 };
+
                 return res(ctx.status(200), ctx.json(projectWithLabel));
             });
 
+            registerApiResponse('EditProject', (req, res, ctx) => {
+                return res(ctx.json(req.body));
+            });
+
```

Comment:
Just curious: why was this needed? I guess with this we make sure that the UI shows the updated labels in the tests?

Reply:
Good question. The change is that, now, instead of reloading the page, i update the metadata onSuccess [here](https://github.com/open-edge-platform/geti/pull/280/files#diff-9b87f4e1dc5df5e0fbd9e8f6c0d4c38abe4ff9a97ad9266f5f061667e3f4fe11R125) . So, we not only get 1 less reload, but we also get an "optimistic update". We PUT, and then GET. But what we get is the same information as the PUT's payload. So the change is instant and we refresh 1 less time :)